recipes/antspyx/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "antspyx" %}
+{% set version = "0.5.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/antspyx-{{ version }}.tar.gz
+  sha256: 893e5b45175e278c1e5ab2c4c663f4dbcb08098ee842ba35eca73d785f263749
+  patches:
+    - itk_patch.diff
+
+build:
+  noarch: python
```

Comment:
Please read the documentation about noarch packages. You cannot use compilers in a noarch recipe. Therefore, this is not a noarch package.